host/libvirt_lxc_backend.go

```diff
@@ -665,6 +698,18 @@ func (c *libvirtContainer) watch(ready chan<- error) error {
 	}
 	defer c.Client.Close()
 
+	go func() {
+		// Workaround for mounts leaking into the libvirt_lxc supervisor process,
+		// see https://github.com/flynn/flynn/issues/1125 for details. Remove
+		// nsumount from the tree when deleting.
+		g.Log(grohl.Data{"at": "cleanup_mounts", "status": "start"})
+		if err := c.cleanupMounts(c.domain.ID); err != nil {
+			g.Log(grohl.Data{"at": "cleanup_mounts", "status": "error", "err": err})
+		}
+		c.unbindMounts()
```

Comment:
It took me a minute to realise why we are immediately unbinding all the mounts, so I think this could do with a comment that by this point, the mount namespace has been cloned and these are no longer necessary in the root namespace.